cluster/ci/monitoring/mixins/grafana_dashboards/ghproxy.jsonnet

```diff
@@ -159,4 +192,76 @@ dashboard.new(
     x: 20,
     y: 0,
   })
+.addPanel(
+    (graphPanel.new(
+        'Token Usage',
+        description='GitHub token usage by token identifier and API version.',
+        datasource='prometheus',
+        legend_alignAsTable=true,
+        legend_rightSide=true,
+        legend_values=true,
+        legend_current=true,
+        min='0',
+        max='5000',
+    ) + legendConfig)
+    .addTarget(prometheus.target(
+        'label_replace(sum(github_token_usage) by (api_version, token_hash), "token_hash_short", "$1", "token_hash", "([a-z0-9]{5})(.*)")',
+         legendFormat='{{api_version}}:{{token_hash_short}}',
```

Comment:
I would love not to have to do this server-side in prometheus -- is there no string formatting directive to use at runtime?

Reply:
I would like to do that too.
Google did not help me well in this case.
Will ask our friends in monitoring team tomorrow.

Reply:
Not supported yet by grafana:
https://github.com/grafana/grafana/pull/6627